packages/aura-language-server/package.json

```diff
@@ -12,9 +12,8 @@
   "scripts": {
     "clean": "rm -rf lib",
     "cleanTestData": "node ../../scripts/removeTestData.js",
-    "build": "tsc -p . && yarn copy-static-assets",
+    "build": "tsc -p . && node scripts/build.js",
```

Comment:
```suggestion
    "build": "tsc -p . && scripts/build.js",
```
We don't need to specify node with @lcampos 's shebang suggestions (and adding execute permission to the script).

Reply:
I cant take out the node command, because on windows its needed. If i remove it, i will prompt an error with the windows script host. The CircleCI test for windows will never pass, and it will show a "timeout error"(I think because of this error).
This is the error: 
![image](https://user-images.githubusercontent.com/35036353/92748658-1ef6ee80-f364-11ea-855f-974658f6d702.png)

Reply:
Sounds good, yeah windows doesn't support shebang and it's interpreted as JScript. Let's keep as is then.